core/src/main/java/io/questdb/griffin/model/ExpressionNode.java

```diff
@@ -255,4 +255,25 @@ public ExpressionNode newInstance() {
             return new ExpressionNode();
         }
     }
+
```

Comment:
I reckon it'd be good to comment somewhere close to class fields that any new field has to be indcluded here, otherwise behavior might be unpredictable .  It'd be great to write a future-proof test but I'm not sure that's possible .